Graphics_oM/Versioning_42.json

```
@@ -0,0 +1,24 @@
+﻿{
+  "Namespace": {
+    "ToNew": {
+    },
+    "ToOld": {
+    }
+  },
+  "Type": {
+    "ToNew": {
+    "BH.oM.Graphics.GradientOptions": "BH.oM.Graphics.Colours.GradientOptions",
+    "BH.oM.Graphics.GradientCenteringOptions": "BH.oM.Graphics.Enums.GradientCenteringOptions"
+    },
+    "ToOld": {
+    }
```

Could add the ToOld mapping here
```suggestion
    "BH.oM.Graphics.Colours.GradientOptions" : "BH.oM.Graphics.GradientOptions",
    "BH.oM.Graphics.Enums.GradientCenteringOptions" : "BH.oM.Graphics.GradientCenteringOptions"
    }
```